aleph_alpha_client/structured_output.py

```diff
@@ -28,5 +45,37 @@ class JSONSchema:
     def to_json(self) -> Mapping[str, Any]:
         return {"type": "json_schema", "json_schema": asdict(self)}
 
+    @classmethod
+    def from_pydantic(cls, model_class) -> "JSONSchema":
+        """
+        Create a JSONSchema from a Pydantic model class.
+        
+        Parameters:
+            model_class: A Pydantic BaseModel class
+            
+        Returns:
+            JSONSchema instance with the schema generated from the Pydantic model
+            
+        Raises:
+            ValueError: If the provided class is not a Pydantic BaseModel
+        """
+        # Validate that the model_class is a Pydantic BaseModel
+        if not (isinstance(model_class, type) and issubclass(model_class, BaseModel)):
+            raise ValueError(f"Expected a Pydantic BaseModel class, got {type(model_class).__name__}")
```

Comment:
Once we annotate the type (`Type[BaseModel]`) we could remove this explicit check I guess.